packages/core/src/Tab/Tab.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import * as Tabs from '@radix-ui/react-tabs'
+import { TabList } from './Tab.styled'
+import { Box } from '../Box/Box'
+import { TabButton } from './TabButton'
+import { TabChip } from './TabChip'
+import { TabRadio } from './TabRadio'
+import { ButtonChipProps } from '../Chip/ButtonChip/ButtonChip'
+import { IconComponent } from '../Chip/ChipContent/ChipContent'
+
+export type TabDataProps = {
+  id: string
+  icon?: IconComponent
+  text: string
+  onClick?: () => void
+}
+export type TabProps = {
+  orientation?: 'horizontal' | 'vertical'
+  hasHover?: boolean
+  onClick?: () => void
+  size?: 'sm' | 'md' | 'lg' | ['sm' | 'md' | 'lg']
```

Comment:
Currently no implementation for size - wanted to discuss exact sizes with @wesleymartin85